### DISCUSSION TITLE
English translation of this app.

### DISCUSSION THREAD

**Discussion by Jakeverse** [Planshit/Tai#127] [General]
is it possible to make an english version of this software? foreigners like me who are really interested to use this app won't able to recognize this great app. It's really helpful apps for people like me. If anyone able to translate this app then it will be very helpful for english speaking people.

**Comment by noberumotto** [comment]
I will complete this feature as soon as possible.

**Reply by WinsonMok** [reply]
hello, May I ask about the estimate complete time? thanks~

**Reply by noberumotto** [reply]
before April :)
